app/tests/requirements.txt

```diff
@@ -83,7 +83,7 @@ wcwidth==0.2.5 \
 more-itertools==8.12.0 \
     --hash=sha256:43e6dd9942dffd72661a2c4ef383ad7da1e6a3e968a927ad7a6083ab410a688b \
     --hash=sha256:7dc6ad46f05f545f900dd59e8dfb4e84a4827b97b3cfecb175ea0c7d247f6064
-atomicwrites==1.4.0 \
+atomicwrites==1.4.1 \
```

Comment:
The poetry lock changes are expected yes.  For the integration tests, you have to use a tool called [hashin](https://pypi.org/project/hashin/) that's a little tricky to use I can show you and @yashika-khurana how it works this afternoon.